packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/activeDirectories/add/federation-add.html

```diff
@@ -106,7 +106,7 @@
             </oui-field>
             <oui-field
                 class="col-md-6"
-                data-label="{{ 'dedicatedCloud_USER_AD_ADD_field_label_sslThumbprint' | translate }}*"
+                data-label="{{ 'dedicatedCloud_USER_AD_ADD_field_label_sslThumbprint' | translate }}"
```

Comment:
Is it just for UI purpose? Or should the required flag of the input below should be removed?